.cfnlintrc.yaml

```diff
@@ -123,6 +123,7 @@ ignore_templates:
   - tests/translator/output/**/managed_policies_minimal.json  # Intentionally has non-existent managed policy name
   - tests/translator/output/**/function_with_mq.json  # Property "EventSourceArn" can Fn::GetAtt to a resource of types [AWS::DynamoDB::GlobalTable, AWS::DynamoDB::Table, AWS::Kinesis::Stream, AWS::Kinesis::StreamConsumer, AWS::SQS::Queue]
   - tests/translator/output/**/function_with_mq_using_autogen_role.json  # Property "EventSourceArn" can Fn::GetAtt to a resource of types [AWS::DynamoDB::GlobalTable, AWS::DynamoDB::Table, AWS::Kinesis::Stream, AWS::Kinesis::StreamConsumer, AWS::SQS::Queue]
+  - tests/translator/output/**/api_with_propagate_tags.json # Obsolete DependsOn on resource
```

Comment:
Why? Or is it https://github.com/aws/serverless-application-model/issues/1541?

Reply:
Yes, it's this issue. CFN lint gives warning on unnecessary depends on.

Reply:
I can look into this issue as a separate PR. Seems pretty easy to fix.